EazyE2E/ElementHelper/EzTextResult.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace EazyE2E.ElementHelper
+{
+    public class EzTextResult<T>
+    {
+        public bool NotSupported { get; set; }
+        public bool IsMixed { get; set; }
+        public T Result { get; set; }
```

Comment:
Issue is resolved.  Used a constructor.  I'm dumb and didn't think that you could have a constructor for a class that was of a generic type but I was incorrect; this solves the problem.